MAINTAINERS.md

```diff
@@ -20,11 +20,14 @@ This document explains who the maintainers are (see below), what they do in this
 
 ## Current Maintainers
 
-| Maintainer               | GitHub ID                               | Affiliation |
-| ------------------------ | --------------------------------------- | ----------- |
-| Zhongnan Su              | [zhongnansu](https://github.com/zhongnansu)     | Amazon      |
-| Huan Jiang               | [seraphjiang](https://github.com/seraphjiang)   | Amazon      |
-| Kristen Tian             | [kristenTian](https://github.com/kristenTian)   | Amazon      |
+| Maintainer               | GitHub ID                                         | Affiliation |
+|--------------------------|---------------------------------------------------|-------------|
+| Henri Yandell            | [ hyandell ]( https://github.com/hyandell )       | Amazon      |
+| Daniel "dB." Doubrovkine | [ dblock ]( https://github.com/dblock )           | Amazon      |
+| Huy Nguyen               | [ huyaboo ]( https://github.com/huyaboo )         | Amazon      |
+| Zhongnan Su              | [ zhongnansu ]( https://github.com/zhongnansu )   | Amazon      |
+| Huan Jiang               | [ seraphjiang ]( https://github.com/seraphjiang ) | Amazon      |
+| Kristen Tian             | [ kristenTian ]( https://github.com/kristenTian ) | Amazon      |
```

Comment:
let's follow the [guidance](https://github.com/opensearch-project/.github/blob/main/MAINTAINERS.md#becoming-a-maintainer) as @seraphjiang suggested